tests/framework/validators/FileValidatorTest.php

```diff
@@ -452,6 +452,25 @@ public function testValidateAttributeType()
         $this->assertNotFalse(stripos(current($m->getErrors('attr_exe')), 'Only files with these extensions '));
     }
 
+    public function testValidateAttributeDoubleType()
+    {
+        $val = new FileValidator([
+            'extensions' => 'tar.gz, tar.xz',
+            'checkExtensionByMimeType' => false,
```

Comment:
How about tests if `checkExtensionByMimeType` is enabled?

Reply:
mime type for ".gz" - not exists in Yii2 mimeTypes.php

Reply:
ok, i'm add test for other extension from issue - tar.xz